front/lib/strapi/strapiCRUD.ts

```diff
@@ -3,8 +3,8 @@ import qs from 'qs';
 
 const base_url = 'http://localhost:1337/api/';
 
-export const strapiGet = async ({ req_url, filters }: StrapiCRUDType) =>
-    await axios.get(base_url + req_url + '?' + qs.stringify({ filters }));
+export const strapiGet = async ({ req_url, filters, population }: StrapiCRUDType) =>
+    await axios.get(base_url + req_url + '?' + (qs.stringify({ filters }) || population));
```

Comment:
As i see you want to use filters or if you have population you want to skip filters and use population instead?
So i assume that you want to store filters also in population parameter? 
If im right it'd be better to write sth like

`
=> {
if(population) return await axios.get(base_url + req_url + '?' + population + '/' + (qs.stringify({ filters }));
return await axios.get(base_url + req_url + '?' + (qs.stringify({ filters }))
`

Reply:
Or event to try with sth like 
`
=> await axios.get(base_url + req_url + '?' + (population ? population + '/' : undefined) + (qs.stringify({ filters }))
`

but you should thik what idea is simpler one